mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -424,6 +424,37 @@ class ArithOpConversion final : public OpConversionPattern<ArithOp> {
   }
 };
 
+class DivUIOpConversion final : public OpConversionPattern<arith::DivUIOp> {
+public:
+  using OpConversionPattern<arith::DivUIOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(arith::DivUIOp divUiOp,
+                  typename arith::DivUIOp::Adaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Type newRetTy = this->getTypeConverter()->convertType(divUiOp.getType());
+    if (!newRetTy)
+      return rewriter.notifyMatchFailure(divUiOp,
+                                         "converting result type failed");
+    if (!isa_and_nonnull<IntegerType>(newRetTy)) {
+      return rewriter.notifyMatchFailure(divUiOp, "expected integer type");
+    }
+    Type unsignedType = adaptIntegralTypeSignedness(newRetTy, true);
```

Comment:
```suggestion
    Type unsignedType = adaptIntegralTypeSignedness(newRetTy, /*unsigned=*/ true);
```
(I didn't look up the precise argument name)